src/covid19model/models/models.py

```diff
@@ -46,6 +49,39 @@
 plt.rcParams["font.size"] = 15
 plt.rcParams["lines.linewidth"] = 3
 
+def obtainData():
```

Comment:
@twallema 

In order to have these scripts more aligned:
- keep function names lowercase 
- makes sure the function name define also a link with the source of the data instead of the generis  `obtainData`, e.g. get_sciensano_data()`
- keep the data related functions inside the `src/data/...` folder